src/server/graphql/user/user-resolvers.ts

```diff
@@ -0,0 +1,33 @@
+import { Context } from '../../types/context';
+import { Resolvers } from '../types';
+
+export const userResolvers: Resolvers<Context> = {
+	Query: {
+		user: async (_parent, args, context) => {
+			const userModel = await context.dataSources.userDataSource.getItem(args.id);
+			return context.userModelConverter.fromDbModelToGqlModel(userModel);
```

Comment:
What are the advantages of using context variables instead of calling the classes directly?

Reply:
I actually used the context as my dependency injection container. By doing this, instead of calling the classes directly, I have all the benefits of DI, such as flexibility and maintainability. For example, if I wanted to change the configuration of a specific object that is used by resolvers and data sources, I would only need to change it in one place. The same would be possible if I wanted to use a class that inherits from the one that I was serving in the context before.